<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>REInvoiceItem</class>
 <widget class="QDialog" name="REInvoiceItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Invoice Item</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Voucher type</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="EQLineEdit" name="leVaucher">
           <property name="Field" stdset="0">
            <string>f_vaucher</string>
           </property>
          </widget>
         </item>
         <item row="12" column="2">
          <widget class="EQCheckBox" name="chRest">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Available in restaurant</string>
           </property>
           <property name="Field" stdset="0">
            <string>f_rest</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="EQLineEdit" name="leNameEn">
           <property name="Field" stdset="0">
            <string>f_en</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1" colspan="2">
          <widget class="EQLineEdit" name="leTaxName">
           <property name="Field" stdset="0">
            <string>f_taxName</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>No manual charge</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Tax name</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="EQCheckBox" name="chAuto">
           <property name="text">
            <string/>
           </property>
           <property name="Field" stdset="0">
            <string>f_auto</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="EQLineEdit" name="leNameAm">
           <property name="Field" stdset="0">
            <string>f_am</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="2">
          <widget class="EQLineEdit" name="leNameRu">
           <property name="Field" stdset="0">
            <string>f_ru</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1" colspan="2">
          <widget class="EQLineEdit" name="leNoVATDept">
           <property name="Field" stdset="0">
            <string>f_noVatDept</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="EQLineEdit" name="leCode">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="PrimaryKey" stdset="0">
            <bool>true</bool>
           </property>
           <property name="Field" stdset="0">
            <string>f_id</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="EQLineEdit" name="leGroupName">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="EQLineEdit" name="leGroupCode">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="Field" stdset="0">
            <string>f_group</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Code</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="EQLineEdit" name="lePrice">
           <property name="Field" stdset="0">
            <string>f_price</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Group</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Tax VAT dept</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>ADGT code</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Price</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1" colspan="2">
          <widget class="EQLineEdit" name="leVATDept">
           <property name="Field" stdset="0">
            <string>f_vatDept</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1" colspan="2">
          <widget class="EQLineEdit" name="leADGT">
           <property name="Field" stdset="0">
            <string>f_adgt</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Tax No VAT dept</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Name, am</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Name, en</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Name, ru</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Tax, reception</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="EQLineEdit" name="leReceptionTax"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tax Print</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="EQTableWidget" name="tblTaxPrint">
         <column>
          <property name="text">
           <string>Computer</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Department</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>57</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOk">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>56</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EQCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">eqcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>EQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">eqtablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
